prow/plank/controller.go

```diff
@@ -564,3 +567,25 @@ func getPodBuildID(pod *coreapi.Pod) string {
 	logrus.Warningf("BUILD_ID was not found in pod %q: streaming logs from deck will not work", pod.ObjectMeta.Name)
 	return ""
 }
+
+func (c *Controller) patchProwjob(srcPJ prowapi.ProwJob, destPJ prowapi.ProwJob) error {
+	srcPJData, err := json.Marshal(srcPJ)
+	if err != nil {
+		return fmt.Errorf("cannot create JSON patch: error marshal source prowjob: %v", err)
```

Comment:
This seems excessively long, what additional info does this include beyond: `"marshal srcPJ: %v", err`?

Reply:
https://github.com/kubernetes/test-infra/pull/13420#discussion_r303053156

/shrug

Reply:
"marshal source prow job: %v"

Reply:
done